plugins/hypervisors/kvm/src/com/cloud/hypervisor/kvm/resource/KVMHAMonitor.java

```diff
@@ -115,22 +115,33 @@ protected void runInContext() {
                     }
 
                     String result = null;
-                    for (int i = 0; i < 5; i++) {
+                    // Try 3 times, but sleep in between tries to ensure it isn't a short lived transient error
+                    for (int i = 1; i <= 4; i++) {
```

Comment:
the comment says try three times but i count 4 iterations. either the comment or the loop-control needs to change